libsignal-service/src/configuration.rs

```diff
@@ -104,41 +116,39 @@ impl Into<ServiceConfiguration> for SignalServers {
             // configuration with the Signal API staging endpoints
             // see: https://github.com/signalapp/Signal-Desktop/blob/master/config/default.json
             SignalServers::Staging => ServiceConfiguration {
-                service_urls: vec![
-                    "https://textsecure-service-staging.whispersystems.org"
-                        .into(),
-                ],
+                service_url: "https://textsecure-service-staging.whispersystems.org".parse().unwrap(),
+                storage_url:"https://storage-staging.signal.org".parse().unwrap(),
                 cdn_urls: {
                     let mut map = HashMap::new();
-                    map.insert(0, "https://cdn-staging.signal.org".into());
-                    map.insert(2, "https://cdn2-staging.signal.org".into());
+                    map.insert(0, "https://cdn-staging.signal.org".parse().unwrap());
+                    map.insert(2, "https://cdn2-staging.signal.org".parse().unwrap());
                     map
                 },
-                contact_discovery_url: vec![
-                    "https://api-staging.directory.signal.org".into(),
-                ],
+                contact_discovery_url:
+                    "https://api-staging.directory.signal.org".parse().unwrap(),
                 certificate_authority: SIGNAL_ROOT_CA.into(),
                 unidentified_sender_trust_root:
                     "BbqY1DzohE4NUZoVF+L18oUPrK3kILllLEJh2UnPSsEx".into(),
+                zkgroup_server_public_params: bincode::deserialize(&base64::decode("ABSY21VckQcbSXVNCGRYJcfWHiAMZmpTtTELcDmxgdFbtp/bWsSxZdMKzfCp8rvIs8ocCU3B37fT3r4Mi5qAemeGeR2X+/YmOGR5ofui7tD5mDQfstAI9i+4WpMtIe8KC3wU5w3Inq3uNWVmoGtpKndsNfwJrCg0Hd9zmObhypUnSkfYn2ooMOOnBpfdanRtrvetZUayDMSC5iSRcXKpdls=").unwrap()).unwrap(),
```

Comment:
Is there a nicer way to represent the public params? Probably not immediately...

Reply:
Unfortunately not... [`ServerPublicParams` doesn't expose any way to instantiate it without using `serde::Deserialize`](https://github.com/signalapp/zkgroup/blob/master/rust/src/api/server_params.rs#L19)